README.md

```diff
@@ -166,7 +166,7 @@ Example:
 ```sh
 git@github.com:Yin-ka/Sharing-World.git
 ``` -->
```

Comment:
- [ ] Kindly stick to the changes requested by the previous reviewer. You have not made any of the changes at all. Kindly refactor your design to fit the detailed highlights from the reviewer.

  ![image](https://github.com/yin-ka/metrics-web-app/assets/36542986/756adca1-f3c0-426d-a340-c89404591b0f)
  ![image](https://github.com/yin-ka/metrics-web-app/assets/36542986/efefdbf1-b43b-49fb-a102-9ca2306234b7)